src/framework/preferences/html/PrefsEditorTemplate-speak.html

```diff
@@ -1,11 +1,10 @@
 <h2><span class="fl-icon-speak"></span><span class="flc-prefsEditor-speak-label heading-text"></span></h2>
-
+<p class="flc-prefsEditor-speak-choice-label"></p>
 <div class="fl-prefsEditor-onoff">
     <input type="checkbox" id="speak-choice" class="flc-prefsEditor-speak" />
     <label for="speak-choice">
         <span class="fl-prefsEditor-switch" data-checkboxStateOn="ON" data-checkboxStateOff="OFF">
             <span class="fl-prefsEditor-switch-inner"></span>
         </span>
-        <span class="flc-prefsEditor-speak-choice-label"></span>
```

Comment:
The problem with this change is that there is no longer a label for the checkbox. When an assistive technology comes to the checkbox it will only read its state (checked/unchecked) but not what it is for. We need to maintain a label.